oem.json

```diff
@@ -4250,15 +4753,23 @@
     },
     {
         "id": "com.oppo.webview",
-        "description": "Oppo Webview\nA WebView is a system component for the Android operating system (OS) that allows Android apps to display content \nfrom the web directly inside an application. It's based on Chrome.\nWARNING: Make to have another Webview before uninstalling it or some apps may not work properly\nOn open-source privacy oriented Webview is Bromite (https://www.bromite.org/system_web_view)\n",
+        "label": "Oppo Webview",
```

Comment:
Source?

Reply:
According the user report i.e from `description`.